action/api/src/main/java/io/knotx/fragments/action/api/Cacheable.java

```diff
@@ -21,7 +21,7 @@
 import java.lang.annotation.Target;
 
 /**
- * Marks {@link ActionFactory} as one delivering cacheable actions.
+ * Flags {@link ActionFactory} that produces a stateful action.
```

Comment:
If it produces stateful action, why not `@Stateful` 😉  ? It might be confusing for devs.